src/main/java/org/ibp/api/rest/analysis/SiteAnalysisResource.java

```diff
@@ -0,0 +1,36 @@
+package org.ibp.api.rest.analysis;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.generationcp.middleware.service.impl.analysis.MeansImportRequest;
+import org.ibp.api.java.analysis.SiteAnalysisService;
+import org.ibp.api.rest.dataset.DatasetDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Api(value = "Site Analysis Service")
+@Controller
+public class SiteAnalysisResource {
+
+	@Autowired
+	private SiteAnalysisService siteAnalysisService;
+
+	@ApiOperation(value = "Create means dataset", notes = "Create means dataset")
+	@RequestMapping(value = "/crops/{cropName}/programs/{programUUID}/studies/{studyId}/means-data", method = RequestMethod.POST)
```

Comment:
Hi @abatac quick comment on this, I suggest we use `/crops/{cropName}/programs/{programUUID}/studies/{studyId}/datasets/means` because what we are really creating the dataset at this point, not only data for it. Also it is more consistent with other resources we have implemented 
CC @darla-leafnode